server/server.js

```diff
@@ -20,8 +25,74 @@ app.use(express.static(__dirname + '/../public/'));
 // app.get('/api/acct/signedin', acctController.checkAuth);
 // app.get('/api/acct/logout', acctController.checkAuth);
 // app.post('/api/acct/addToList', acctController.addToList);
+
+
+
+
+
+// configuration ===========================================
+
+// Passport Facebook strategy configuration=================
+
+// OAuth 2.0-based strategies require a `verify` function which receives the
+// credential (`accessToken`) for accessing the Facebook API on the user's
+// behalf, along with the user's profile.  The function must invoke `cb`
+// with a user object, which will be set at `req.user` in route handlers after
+// authentication.
+var userController = require('./users/userController');
```

Comment:
var userController should probably be defined at the top of the file, with the other require statements.